README.md

```diff
@@ -270,6 +270,12 @@ In order to run tests with coverage run:
 
 ```npm run coverage```
 
+## Contributing
+
+### Config
+
+Values in the config files get their types auto-generated by [node-config-ts](https://github.com/tusharmath/node-config-ts) when running `npm run postinstall`.
+
 ## Maintainers
```

Comment:
Can we finally change this lol? It bugs me everytime I come across it. I'd say @stbrody and @stephhuynh18 should probably be here instead.

Reply:
haha yeah I will finalize a pr with changes for this

Reply:
see https://github.com/ceramicnetwork/ceramic-anchor-service/pull/711